docs/_articles/en/soql/soql-builder.md

```diff
@@ -27,30 +27,38 @@ Next, install the SOQL Builder extension from [Visual Studio Marketplace](https:
 
 ## Launch SOQL Builder
 
-Launch SOQL Builder from within a Salesforce DX project.
+Launch SOQL Builder from within a Salesforce DX project. How you launch SOQL Builder depends on if you have an existing `.soql` file or if you plan to create one.
 
 ### Prerequisites
 
 - In VS Code, open a Salesforce DX project.
 - Authorize the org whose objects you want to query.
 
+### Open an Existing SOQL File in SOQL Builder
+
 DX projects have a sample `accounts.soql` file in the `<project-folder>/scripts/soql` directory. However, you can create and store your `.soql` files in any directory.
 
 1. (If necessary) Create a `.soql` file.
 1. Right-click the file name, select **Open With**, then **SOQL Builder**.
 
 ![Right-click to open SOQL Builder](./images/soql-builder-open.gif)
 
+### Launch SOQL Builder and Create a Query
+
+1. From the command palette, run **SFDX: Create Query in SOQL Builder**.
+1. Click **File > Save** to save the query. Make sure to retain the `.soql` file extension.
+
 ## Build a Query
 
 As you build your query, watch SOQL Builder display the query syntax while it simultaneously updates the `.soql` file. After you’re done building your statements, click **Run Query** to view the output.
 
 ![Build your query](./images/soql-builder.gif)
 
+**Tip:** If using the text editor to build your query, you can validate your syntax by turning on the SOQL Editor Remote Checks setting.
```

Comment:
@dobladez - added a tip here regarding your syntax validation setting.

Reply:
Great.  Sorry I missed your comment earlier.
Same facts:
* Basic syntax validation always happens as the user types. This extra option if for additional (deeper and semantic) checks, including validation of object/fields in the org
* The feature is experimental and reports incorrect errors for certain queries
* Given that this feature requires making regular calls to the server as the user types, it may not perform well on slow connections